app/controllers/tutorials_controller.rb

```diff
@@ -74,4 +74,11 @@ def set_tutorial
     def tutorial_params
       params.require(:tutorial).permit(:title, :tagline, :content)
     end
+
+    def require_login
```

Comment:
Check Devise for a helper method that does this so you don't need to define it yourself.  If you want to keep your method definition, move this logic to the application controller so it's not duplicated in the codebase.